src/main/java/com/stripe/net/StripeResponse.java

```diff
@@ -7,47 +7,43 @@
 
 	int responseCode;
 	String responseBody;
-	Map<String, List<String>> responseHeaders;
+	StripeHeaders headers;
 
 	public StripeResponse(int responseCode, String responseBody) {
 		this.responseCode = responseCode;
 		this.responseBody = responseBody;
```

Comment:
Do you mind renaming `responseCode` and `responseBody` to just `code` and `body`?